ivy/container/random.py

```diff
@@ -1039,6 +1039,24 @@ def static_shuffle(
         -------
         ret
             A container object, shuffled along the first dimension.
+
+        Examples
+        --------
+        >>> x = ivy.Container(a=ivy.array([7,6,0]), \
```

Comment:
Hi, it seems you missed changing the `\` notation in all files as I instructed in my previous review, could you please change these as well? 😅